src/popup.html

```diff
@@ -16,8 +19,7 @@ <h1 data-localize="appName"></h1>
     </header>
 
     <div class="search">
-      <label for="my-search" data-localize="search"></label>
-      <input type="search" id="my-search" name="search" autocomplete="off">
+      <input type="search" id="my-search" name="search" placeholder="Search" autocomplete="off">
```

Comment:
Nice I like the update.

Reply:
We need to make changes though so search is translated correctly in each language.